config/ndlar/ndlar_full_chain_flash_nersc_240819.cfg

```diff
@@ -0,0 +1,499 @@
+          particle_event: particle_pcluster
+          neutrino_event: neutrino_mc_truth
+          cluster_event: cluster3d_pcluster
+          skip_empty: true
+        neutrinos:
+          parser: neutrino
+          neutrino_event: neutrino_mc_truth
+          cluster_event: cluster3d_pcluster
+        meta:
+          parser: meta
+          sparse_event: sparse3d_pcluster
+        run_info:
+          parser: run_info
+          sparse_event: sparse3d_pcluster
+        trigger:
+          parser: trigger
+          trigger_event: trigger_base
+        flashes:
+          parser: flash
+          flash_event: opflash_light
+
+  writer:
+    name: hdf5
+    file_name: null
+    overwrite: true
+    keys:
+      - run_info
+      - meta
+      - trigger
+      - flashes
+      - points
+      - points_label
+      - depositions
+      - depositions_label
+      - reco_particles
+      - truth_particles
+      - reco_interactions
+      - truth_interactions
+
+# Model configuration
+model:
+  name: full_chain
+  weight_path: /global/cfs/cdirs/dune/www/data/2x2/simulation/mlreco_weights/2x2_240819_snapshot.ckpt
+  to_numpy: true
+
+  network_input:
+    data: data
+    seg_label: seg_label
+    clust_label: clust_label
+
+  loss_input:
+    seg_label: seg_label
+    ppn_label: ppn_label
+    clust_label: clust_label
+    coord_label: coord_label
+
+  modules:
+    # General chain configuration
+    chain:
+      deghosting: null
+      charge_rescaling: null
+      segmentation: uresnet
+      point_proposal: ppn
+      fragmentation: graph_spice
+      shower_aggregation: grappa
+      shower_primary: grappa
+      track_aggregation: grappa
+      particle_aggregation: null
+      inter_aggregation: grappa
+      particle_identification: grappa
+      primary_identification: grappa
+      orientation_identification: grappa
+      calibration: null
+
+    # Semantic segmentation + point proposal
+    uresnet_ppn:
+      uresnet:
+        num_input: 1
+        num_classes: 5
+        filters: 32
+        depth: 5
+        reps: 2
+        allow_bias: false
+        activation:
+          name: lrelu
+          negative_slope: 0.33
+        norm_layer:
+          name: batch_norm
+          eps: 0.0001
+          momentum: 0.01
+  
+      ppn:
+        classify_endpoints: false
+  
+    uresnet_ppn_loss:
+      uresnet_loss:
+        balance_loss: false
+  
+      ppn_loss:
+        mask_loss: CE
+        resolution: 5.0
+
+    # Dense clustering
+    graph_spice:
+      shapes: [shower, track, michel, delta]
+      use_raw_features: true
+      invert: true
+      make_clusters: true
+      embedder:
+        spatial_embedding_dim: 3
+        feature_embedding_dim: 16
+        occupancy_mode: softplus
+        covariance_mode: softplus
+        uresnet:
+          num_input: 4 # 1 feature + 3 normalized coords
+          filters: 32
+          input_kernel: 5
+          depth: 5
+          reps: 2
+          spatial_size: 31231
```

Comment:
This is going to be an issue in that this `spatial_size` is used to scale the coordinate input into the Graph-SPICE network to values between -1 and 1. It will still produce numbers between -1 and 1 but the metric no longer means the same in that space (i.e. a displacement of 0.1 after scaling is not the same in 2x2 and ND-LAr). Not asking to change anything but this is a caveat to say I don't expect Graph-SPICE to work without being retrained on an image of the appropriate size. It would also be worth investigating if we need to explicitly input these coordinates at all or not, I can take a look at that.